ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_extrema_finding.py

```diff
@@ -344,3 +344,45 @@ def test_numpy_fmax(
         out=None,
         where=where,
     )
+
+
+# fmin
+@handle_frontend_test(
+    fn_tree="numpy.fmin",
+    dtype_and_inputs=helpers.dtype_and_values(
```

Comment:
this should resemble something like this (please alter according to the needs of the function):
```
dtypes_values_casting=np_frontend_helpers.dtypes_values_casting_dtype(
         arr_func=[
             lambda: helpers.dtype_and_values(
                 available_dtypes=helpers.get_dtypes("float"),
             )
         ],
         get_dtypes_kind="float",
     ),
```